### ISSUE TITLE
Ignore directories

### ISSUE THREAD

**Issue by joatmor** [koalaman/shellcheck#2905]
#### For bugs
- Rule Id (if any, e.g. SC1000): 
- My shellcheck version (`shellcheck --version` or "online"): 
- [ ] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [ ] I tried on https://www.shellcheck.net/ and verified that this is still a problem on the latest commit

#### For new checks and feature suggestions
- [ ] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/your/interpreter
your script here

```

#### Here's what shellcheck currently says:

.git: .git: openBinaryFile: inappropriate type (is a directory)

#### Here's what I wanted or expected to see:

Shellcheck should simply ignore my git directory

**Comment by brother** [comment]
Documentation states that you should supply files on the command lines, not directories. That should have been enough.
